api/services/Search.js

```diff
@@ -155,7 +155,9 @@ module.exports = {
       }
 
       if (opts.autocomplete) {
-        qb.whereRaw('users.name ilike ?', opts.autocomplete + '%')
+        Search.addTermToQueryBuilder(opts.autocomplete, qb, {
+          columns: ['users.name']
+        })
       }
```

Comment:
One alternative which also works fine (though just slightly less performant) is:

```
qb.where(function () {
  this.orWhereRaw("split_part(users.name, ' ', 1) ILIKE ?", opts.autocomplete + '%')
  this.orWhereRaw("split_part(users.name, ' ', 2) ILIKE ?", opts.autocomplete + '%')
  this.orWhereRaw("split_part(users.name, ' ', 3) ILIKE ?", opts.autocomplete + '%')
})

```